src/ParcelRegistry.Api.Oslo/Parcel/List/ParcelListOsloV2Query.cs

```diff
@@ -30,18 +30,23 @@ protected override IQueryable<ParcelDetailV2> Filter(FilteringHeader<ParcelFilte
 
             if (!filtering.ShouldFilter)
             {
-                return parcels;
+                return parcels.Select(x => new ParcelListV2QueryItem
+                {
+                    CaPaKey = x.CaPaKey,
+                    StatusAsString = x.StatusAsString,
+                    VersionTimestampAsDateTimeOffset = x.VersionTimestampAsDateTimeOffset
+                });
             }
 
             if (!string.IsNullOrEmpty(filtering.Filter.AddressId))
             {
                 if (int.TryParse(filtering.Filter.AddressId, out var addressId))
                 {
-                    parcels = parcels.Where(x => x.Addresses.Any(parcelDetailAddress => parcelDetailAddress.AddressPersistentLocalId == addressId));
+                    parcels = parcels.Include(x => x.Addresses).Where(x => x.Addresses.Any(parcelDetailAddress => parcelDetailAddress.AddressPersistentLocalId == addressId));
```

Comment:
Why the include? Is Include not only meant when you want to load the child collection into memory?

Reply:
right, it was something from trial and error :)